commands/reports/report_test.go

```diff
@@ -0,0 +1,63 @@
+package reports
+
+import (
+	"testing"
+
+	"github.com/stampzilla/gozwave/commands"
+	"github.com/stretchr/testify/assert"
+)
+
+type row struct {
```

Comment:
row seems to be used in only TestNew. Maybe put the declaration locally in there insead of out here?